tests/tracer/utils_botocore/test_span_pointers.py

```diff
@@ -0,0 +1,222 @@
+import logging
+import re
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+import mock
+import pytest
+
+from ddtrace._trace._span_pointer import _SpanPointerDescription
+from ddtrace._trace._span_pointer import _SpanPointerDirection
+from ddtrace._trace.utils_botocore.span_pointers import _aws_s3_object_span_pointer_hash
+from ddtrace._trace.utils_botocore.span_pointers import extract_span_pointers_from_successful_botocore_response
+
+
+class TestS3ObjectPointer:
+    class HashingCase(NamedTuple):
+        name: str
+        bucket: str
+        key: str
+        etag: str
+        pointer_hash: str
+
+    @pytest.mark.parametrize(
+        "hashing_case",
+        [
+            # These test cases are copied from the dd-span-pointer-rules
```

Comment:
Let's remove this comment and put a note about this in the description. And maybe make this comment a todo: instead if you'd like?

Reply:
removed!